docker-compose.yml

```diff
@@ -36,26 +45,25 @@ services:
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
     ports:
       - 9200:9200
+      - 9300:9300
     volumes:
       - reference-data-esdata-test:/usr/share/elasticsearch/data
     networks:
-      elastic:
-        aliases:
-          - reference-data-elasticsearch
-          - elastic
-          - elasticsearch
-  kibana:
-        image: docker.elastic.co/kibana/kibana:7.9.3
-        ports:
-            - "5601:5601"
-        networks:
-            - elastic
-        depends_on:
-            - reference-data-elasticsearch
+      - elastic
+
+  #kibana:
```

Comment:
We could do. But it's not doing any harm, and if a developer in future needs easy access to see exactly what data is in the local ES instance this will save them the time of trying to work out what needs to be setup to use kibana locally.